src/main/java/com/group7/krisefikser/config/SecurityConfig.java

```diff
@@ -55,21 +55,19 @@ public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
     source.registerCorsConfiguration("/**", corsConfiguration);
 
     http.cors(cors -> cors.configurationSource(source))
-        .csrf(AbstractHttpConfigurer::disable)
-        .authorizeHttpRequests(authorize -> authorize
-        .requestMatchers(HttpMethod.GET, "/api/affected-area",
-          "/api/point-of-interest", "/h2-console/**").permitAll()
-        .requestMatchers(HttpMethod.POST, "/api/auth/**", "/h2-console/**").permitAll()
-        .requestMatchers(HttpMethod.DELETE, "/api/items/**").hasAnyRole("ADMIN", "SUPER_ADMIN")
-        .requestMatchers("/api/point-of-interest/**", "/api/affected-area/**")
-        .hasAnyRole("SUPER_ADMIN", "ADMIN")
-        .anyRequest().authenticated())
-        .headers(
-          headers -> headers.frameOptions(frameOptionsConfig -> frameOptionsConfig.sameOrigin())
-      )
-        .sessionManagement(session -> session
-        .sessionCreationPolicy(SessionCreationPolicy.STATELESS));
-
+            .csrf(AbstractHttpConfigurer::disable)
+            .authorizeHttpRequests(authorize -> authorize
+                    .requestMatchers(HttpMethod.GET, "/api/affected-area",
+                            "/api/point-of-interest", "/h2-console/**", "api/auth/**").permitAll()

```

Comment:
Shoulnt there be a '/' before api?
```suggestion
                            "/api/point-of-interest", "/h2-console/**", "/api/auth/**").permitAll()

```